src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -163,46 +174,42 @@ namespace WindowCovering {
 
 bool IsOpen(chip::EndpointId endpoint)
 {
-    uint16_t liftPosition = 0;
-    uint16_t liftLimit    = 0;
-    uint16_t tiltPosition = 0;
-    uint16_t tiltLimit    = 0;
-    bool isOpen           = false;
+    EmberAfStatus liftStatus;
+    app::DataModel::Nullable<Percent100ths> liftPosition;
 
-    if (HasFeature(endpoint, WindowCoveringFeature::kLift) && HasFeature(endpoint, WindowCoveringFeature::kPositionAwareLift) &&
-        HasFeature(endpoint, WindowCoveringFeature::kAbsolutePosition) &&
-        EMBER_ZCL_STATUS_SUCCESS == Attributes::TargetPositionLiftPercent100ths::Get(endpoint, &liftPosition) &&
-        EMBER_ZCL_STATUS_SUCCESS == Attributes::InstalledOpenLimitLift::Get(endpoint, &liftLimit))
-    {
-        isOpen = liftPosition == liftLimit;
-    }
-    else if (HasFeature(endpoint, WindowCoveringFeature::kLift) &&
-             HasFeature(endpoint, WindowCoveringFeature::kPositionAwareLift) &&
-             EMBER_ZCL_STATUS_SUCCESS == Attributes::TargetPositionLiftPercent100ths::Get(endpoint, &liftPosition))
-    {
-        isOpen = 0 == liftPosition;
-    }
+    EmberAfStatus tiltStatus;
+    app::DataModel::Nullable<Percent100ths> tiltPosition;
 
-    if (HasFeature(endpoint, WindowCoveringFeature::kTilt) && HasFeature(endpoint, WindowCoveringFeature::kPositionAwareTilt) &&
-        HasFeature(endpoint, WindowCoveringFeature::kAbsolutePosition) &&
-        EMBER_ZCL_STATUS_SUCCESS == Attributes::TargetPositionTiltPercent100ths::Get(endpoint, &tiltPosition) &&
-        EMBER_ZCL_STATUS_SUCCESS == Attributes::InstalledOpenLimitTilt::Get(endpoint, &tiltLimit))
-    {
-        isOpen = isOpen && tiltPosition == tiltLimit;
-    }
-    else if (HasFeature(endpoint, WindowCoveringFeature::kTilt) &&
-             HasFeature(endpoint, WindowCoveringFeature::kPositionAwareTilt) &&
-             EMBER_ZCL_STATUS_SUCCESS == Attributes::TargetPositionTiltPercent100ths::Get(endpoint, &tiltPosition))
-    {
-        isOpen = isOpen && 0 == tiltPosition;
-    }
+    liftStatus = Attributes::TargetPositionLiftPercent100ths::Get(endpoint, liftPosition);
+    tiltStatus = Attributes::TargetPositionTiltPercent100ths::Get(endpoint, tiltPosition);
 
-    return isOpen;
+    if ((liftStatus != EMBER_ZCL_STATUS_SUCCESS) || liftPosition.IsNull())
+        return false;
+
+    if ((tiltStatus != EMBER_ZCL_STATUS_SUCCESS) || tiltPosition.IsNull())
+        return false;
+
+    return ((liftPosition.Value() == WC_PERCENT100THS_MIN) && (tiltPosition.Value() == WC_PERCENT100THS_MIN));
```

Comment:
So this will return false if only one of lift and tilt is supported on the endpoint?  That seems very odd...

Reply:
These helpers are totally changed later... 
This is why i did the bear minimum for this demo
But ok i ll update

Reply:
I have different implementation in later PR
For now i simply split the feature in 2 for this PR